rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -251,7 +252,6 @@ SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 		chbox_gs_overlay->Enable(false);
 	}
 
-	cbox_gs_render->Append("Vulkan");
```

Comment:
OSX doesn't support Vulkan.
Currently the windowing system support is missing for Xorg/Wayland in Vulkan backend. It can be fixed if someone can test it...